packages/frontend/pages/positions.tsx

```diff
@@ -299,8 +308,13 @@ export function Positions() {
                   ) : (
                     <>
                       <Typography variant="body1" className={shortGain.isLessThan(0) ? classes.red : classes.green}>
-                        ${wethAmount.minus(buyQuote).times(toTokenAmount(index, 18).sqrt()).toFixed(2)} (
-                        {wethAmount.minus(buyQuote).toFixed(5)} ETH)
+                        $
+                        {wethAmount
+                          .minus(buyQuote)
+                          .times(toTokenAmount(index, 18).sqrt())
+                          .plus(lastDeposit?.ethAmount.times(lastDeposit?.ethPriceAtDeposit.minus(ethPrice)))
```

Comment:
Ah you might need to address this in the next PR about unrealized PnL, but if someone partially closes this prob won't account for that 

Good for now for this PR since this one is focused on adding in ETH collateral, but just a heads up there